src/lib/fastAuthWalletConnection.ts

```diff
@@ -302,30 +322,81 @@ export class FastAuthWalletConnection {
     );
     newUrl.searchParams.set('success_url', callbackUrl || currentUrl.href);
     newUrl.searchParams.set('failure_url', callbackUrl || currentUrl.href);
```

Comment:
Very nice but with time these params can be removed for simplicity so that we have iframe only logic 👍

Reply:
Yeah, let's open a ticket for this clean up! Will be amazing to remove the duplicated logic